### DISCUSSION TITLE
Change install options?

### DISCUSSION THREAD

**Discussion by dabrahams** [Homebrew/discussions#820] [Everyday usage] [answered]
Just curious: suppose I 

```
brew install emacs-plus --with-xwidgets --with-mailutils
```

but I want to undo the xwidgets option and end up as though I had

```
brew install emacs-plus --with-mailutils
```

in the first place.  Is there a way to do that without `brew uninstall`ing `emacs-plus` first?

Thanks,

**Comment by SMillerDev** [comment, marked as answer]
No, if you want to change install options you need to install again.
